src/servicebus/azext_servicebus/commands.py

```diff
@@ -0,0 +1,127 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.sdk.util import CliCommandType
+from azure.cli.command_modules.servicebus._client_factory import (namespaces_mgmt_client_factory,
+                                                                  queues_mgmt_client_factory,
+                                                                  topics_mgmt_client_factory,
+                                                                  subscriptions_mgmt_client_factory,
+                                                                  rules_mgmt_client_factory,
+                                                                  disaster_recovery_mgmt_client_factory,)
+
+
+
+def load_command_table(self, _):
+    sb_namespace_util = CliCommandType(
+        operations_tmpl='azure.mgmt.servicebus.operations.namespaces_operations#NamespacesOperations.{}',
```

Comment:
The operations paths are wrong.
It should be `azext_servicebus.servicebus....` since the SDK is in the extension.

I'd be surprised if this worked as expected after a user installs the extension.

Reply:
Same below...